lfpse.R

```diff
@@ -6,62 +6,64 @@ if(lfpse_categorical==0){
   lfpse_categorical <- expr(1==1)
 }
 
-# reading reference tables
-QuestionReference <- tbl(con_lfpse, "QuestionReference") |> collect()
-ResponseReference <- tbl(con_lfpse, "ResponseReference") |> collect()
-
+#set latest revision table from events
+latest_revision_table <- tbl(con_lfpse, in_schema("analysis", "Events")) |>
+  group_by(Reference) |>
+  summarise(Revision = max(Revision), #i.e., the most up to date version of a record 
+            reported_date = min(SubmissionDate)

```

Comment:
This is fine but worth checking with Fran's team if we need to retain Revision Date as something like 'date last updated'?

Reply:
Dropped a line to Fran on teams. If they want us to add, another name could be 'date last revised'.

Reply:
Agreed with JM we can add later if necessary